src/database/Database.cpp

```diff
@@ -119,6 +119,7 @@ Database::applySchemaUpgrade(unsigned long vers)
 
     case 4:
         BanManager::dropAll(*this);
+        DataFrame::dropAll(*this); // not and error, it changed again
```

Comment:
I'll check if the lastmodified fiels was stored as '0' in XDR files.
It is possible that it was only zeroed in database, but this still needs to be checked.